006-admin-server.md

```diff
@@ -0,0 +1,116 @@
+# Strimzi Admin Server
+
+The Strimzi Admin repository was setup in December 2019 to hold an implementation of an Admin API. The repository has not had any contributions yet so this proposal sets out what the structure of an Admin Server might look like in order to get to the initial implementation of the server.
+
+## Motivation
+
+An API server will:
+* provide a consolidated backend API for additional interfaces like a browser based UI or a CLI.
+* be capable of supporting both REST and GraphQL interfaces.
+* allow for more sophisticated APIs to be built on top of the existing APIs like the kubernetes API providing value-add to existing content.
+
+## Proposal
+
+### The server
+The proposal is to make a modular server using the Vert.x-Web toolkit which has strong support for creating both REST and GraphQL interfaces.
+The server contains a single Vert.x verticle which listens for inbound traffic on 1 or more configurable ports.
+Allowing multiple configurable ports will allow the server to expose different functionality on each port, the simplest example of this being exposing an HTTP API on one port and a separate HTTPS API on a different port.
```

Comment:
I assume different functionality includes varying methods of authn and authz too? i.e. a mechanism to set different authed endpoints, one scram, one TLS etc?

Or is this handled by the security context piece in the **Security** section?

Reply:
I have rewritten this section to clarify the uses and have specifically cited the use of multiple authentication schemes as the main driver for needing to do this.